en/core-libraries/form.rst

```diff
@@ -98,6 +98,42 @@ accordingly. We could have also used the ``validate()`` method to only validate
 the request data::
 
     $isValid = $form->validate($this->request->data);
+    
+Setting Form Values
+===================
+
+In order to set the values for the fields of a modelless form, you can define
```

Comment:
Context will remain context, but the smart peeps will generalize it and try it on other types of forms when it's scope isn't limited ;)